test/it/fixtures/packages/module-usage/expected.txt

```diff
@@ -45,11 +45,11 @@
 
   3) TimoutSuite
        slow:
-     Error: Timeout of 20ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.
+     Error: Timeout of 100ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.
 
   4) TimeoutSuite2
        slow:
-     Error: Timeout of 20ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.
+     Error: Timeout of 100ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.
```

Comment:
By the look of it at some point we will template these with a library similar to mustache.

Reply:
Good point, already thought of this, too.